vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,324 @@
+                                      annotation=annotation_file,
+                                      global_qvalue_cutoff = 0.01,
+                                      qvalue_cutoff = 0.01,
+                                      pg_qvalue_cutoff = 0.01,
+                                      useUniquePeptide = TRUE,
+                                      removeFewMeasurements = TRUE,
+                                      removeOxidationMpeptides = TRUE,
+                                      removeProtein_with1Feature = TRUE,
+                                      MBR=TRUE)
+```
+
+```{r DIANNtoMSstatsFormat head}
+head(msstats_format)
+```
+
+
+
+### __1.3 Loading PD Data to MSstats__
+
+The imported data from Step 1.1. now must be converted through MSStatsConvert package's PDtoMSstatsFormat converter.
+
+This function converts the Proteome discoverer output into the required input format for MSstats.
+
+```
+MSstatsConvert::PDtoMSstatsFormat(input,
+                              annotation,
+                              useNumProteinsColumn=FALSE,
+                              useUniquePeptide=TRUE,
+                              summaryforMultipleRows=max,
+                              fewMeasurements="remove",
+                              removeOxidationMpeptides=FALSE,
+                              removeProtein_with1Peptide=FALSE,
+                              which.quantification = 'Precursor.Area',
+                              which.proteinid = 'Protein.Group.Accessions',
+                              which.sequence = 'Sequence')
+```
+
+Actual Data modification can be seen below:
+
+```{r code PDtoMSstatsFormat}
+
+pd_imported = MSstatsConvert::PDtoMSstatsFormat(pd, annotation, use_log_file = FALSE)
+
+head(pd_imported)
+```
+
+### __1.4 Data Process__
+
+Once we import the dataset correctly with Formatter, we need to pre-process the data which is taken care by dataProcess.
+
+This step is essentially Data pre-processing and quality control of MS runs of the original raw data into quantitative data for model fitting and group comparison.
+
+There are 3 main steps that happen in background :
+
+* __Normalization__ - There are three different normalizations supported. 'equalizeMedians' (default) represents constant normalization (equalizing the medians) based on reference signals is performed. 'quantile' represents quantile normalization based on reference signals is performed. 'globalStandards' represents normalization with global standards proteins. FALSE represents no normalization is performed.
+
+* __Feature selection__ - This also has three options i.e. Select All features, Top-N features (by mean intensity) or “Best” features.
+* __Missing value imputation__ - We impute plausible values in case of missing data points. The RunLevelData can be queried to show Number of imputed intensities (censored intensities) in a RUN and Protein.
+
+
+``` {r code dataProcess}
+summarized = dataProcess(pd_imported)
```

Comment:
I think this is where we want to play around with different normalization / imputation / feature selection algorithms and show how the corresponding plots will look.  So instead of separating out data processing into `dataProcess` and `dataProcessPlots` subsections, I think what would be better is separating out into subsections like `normalization`, `imputation`, `feature selection`.  And then at each stage, show how the plots look different from before.

Reply:
This is fine for now, but let's keep this in mind in the future as you learn more about how MSstats works.